### ISSUE TITLE
[BUG] Groups - Add image - Camera mislabeled as "Gallery"

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5766]
**Describe the bug**
I believe it is an error that the way to upload via Camera is marked “Gallery” here. This is causing the same violation as in #4874.

**To Reproduce**
Using TalkBack or VoiceOver
1. Go to a Group that user is a member of
2. Click on '+’
3. Select ‘Create Post’
4. Select ‘Add Image’

**Expected behavior**
The options include Camera, Gallery, or Cancel.

**Actual behavior**
The options include Gallery, Gallery, or Cancel. (The first gallery opens the camera and offers a close camera button.)

**Screenshots**

![Image](https://github.com/user-attachments/assets/20ea6e20-f373-456e-b7f2-ce2f8612f606)

**Smartphone (please complete the following information):**
- Device: Google Pixel 8a
 - OS:  Android 16
 - Version 7.3.51

 - Device: iPhone 14+
 - OS:  18.6.2
 - Version 7.3.51

**Additional context**

**Comment by acovher2** [comment]
@vburgett This is not an Accessibility bug solely. It will affect all users, so I marked it as High. Feel free to re-prioritize if needed.

**Comment by dobromirdobrev** [comment]
Fixed in version 7.3.52 / 8.0.8

**Comment by acovher2** [comment]
Verified as fixed in 7.3.52

**Comment by acovher2** [comment]
![Image](https://github.com/user-attachments/assets/a2a34201-2521-4593-9a56-2a0ada655335)
